src/ddmd/cond.d

```diff
@@ -508,12 +508,20 @@ extern (C++) final class StaticIfCondition : Condition
                 printf("\ts = '%s', kind = %s\n", sds.toChars(), sds.kind());
             }
         }
+
+        int errorReturn()
```

Comment:
No, because `inc` is a member of `Condition`, which `StaticIfCondition` inherits. The error message is:
```
Error: need 'this' for 'inc' of type 'int'
```